templates/pyramid/files/.platform.app.yaml

```diff
@@ -0,0 +1,35 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses.
+type: "python:3.7"
+
+# The build-time dependencies of the app.
+dependencies:
+    python:
+        pipenv: "2018.10.13"
+
+# The hooks executed at various points in the lifecycle of the application.
+hooks:
+    build: |
+      pipenv install --system --deploy
+# The size of the persistent disk of the application (in MB).
+disk: 1024
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    mysql: "mysql:mysql"
```

Comment:
Almost everywhere else we call the main DB relationship `database`.  Let's stick to that.